CHANGELOG.md

```diff
@@ -15,9 +15,14 @@ Changes are grouped as follows
 
 
 
-## [0.94.4] - 2024-05-24
+## [0.94.4] - 2024-05-28
 ### Changed
 * FDM v1 resync: Supporting default values in data_model_configuration also without specifying source
+### Fixed
+* FDM v1 resync:
+  * Removing hardcoded reference to files/model.yaml, using all source_files in data_model_configuration instead
+  * Handling empty files
+  * Handling "missing" penstock_loss_factors and "missing" subtype lists
```

Comment:
**note:** this is the wrong version to use as reference, the branch you're merging into hasn't done a version bump, so 0.94.4 is what's in `main`

Reply:
Yes, will fix/update before merging into main (but may merge it as it is now into resync-data-update-mappings)